CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ ycm_default_dirs(${VARS_PREFIX})
 option(CODYCO_USES_URDFDOM "Compile yarpWholeBodyInterface with URDF support" TRUE)
 option(COMPILE_AS_SHARED_LIBRARY "Compile ${PROJECT_NAME} as a shared library" TRUE)
 
-option(ENABLE_TESTS "Enable unit testing" FALSE)
+option(YARPWHOLEBODYINTERFACE_ENABLE_TESTS "Enable unit testing" FALSE)
```

Comment:
The `VARS_PREFIX` CMake variable is there exactly for that (if you don't like its current value we can change it, but I'll prefer to avoid duplicated prefixes).